gms/api/src/main/pegasus/com/linkedin/dataset/Dataset.pdl

```diff
@@ -61,11 +61,6 @@ record Dataset includes DatasetKey, ChangeAuditStamps, VersionTag {
    */
   uri: optional Uri
 
-  /**
-   * List of dataset instances for this dataset
-   */
-  deploymentInfos: array[DeploymentInfo] = [ ]
```

Comment:
Is this backwards incompatible?

Reply:
It is, luckily no one is producing nor consuming the field so should be fine.

Reply:
Are we sure about that? I'll approve this, but it seems like we maybe need a better stand around breaking changes.

Maybe we should add an experimental branch or something? Then we reserve the right to break it whenever we want. :p 

Granted, might be hard to manage an experimental branch and keep it in sync with master. idk?

Reply:
There's a reason why we're on 0.x :)